src/main/groovy/com/marklogic/gradle/task/MlcpTask.groovy

```diff
@@ -329,4 +340,21 @@ class MlcpTask extends JavaExec {
 		mlcpBean.setUri_id(uri_id)
 		mlcpBean.setXml_repair_level(xml_repair_level)
 	}
+
+	List<String> loadOptionsFileFields(options_file) {
+		def optionsFileFields = []
+		try {
+			new File(options_file).eachLine { line ->
+				if (line.startsWith("-")) {
+					optionsFileFields.add(line.substring(1))
+				}
+			}
+		} catch(Exception e) {
+			getLogger().error("Unable to read the specified options file. Not passing '-host', '-username', or '-password' to MLCP since they may exist in the options file.")
```

Comment:
Just do "warn" for this as it's a warning, no error is being thrown. And include a `"; cause: " + e.getMessage()` so that the user can see what the error was.